subsys/nrf_security/src/drivers/cracen/cracenpsa/src/cracen_mac_cmac.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <psa/crypto.h>
+#include "common.h"
+#include "cracen_psa_primitives.h"
+
+/**
+ * @brief Set up a CMAC operation.
+ *
+ * This function initializes a CMAC operation with the provided key attributes
+ * and key buffer. It prepares the operation structure for subsequent CMAC
+ * processing.
+ *
+ * @param[in,out] operation        Pointer to the CMAC operation structure to be initialized.
+ * @param[in]     attributes       Pointer to the key attributes structure.
+ * @param[in]     key_buffer       Pointer to the buffer containing the key material.
+ * @param[in]     key_buffer_size  Size of the key buffer in bytes.
+ *
+ * @return PSA_SUCCESS on success, or an appropriate error code on failure.
+ */
+psa_status_t cracen_cmac_setup(cracen_mac_operation_t *operation,
+			       const psa_key_attributes_t *attributes, const uint8_t *key_buffer,
+			       size_t key_buffer_size);
+
+/**
+ * @brief Update a CMAC operation with input data.
+ *
+ * This function processes a chunk of input data as part of a CMAC operation.
+ * It can be called multiple times to process data in chunks.
+ *
+ * @param[in,out] operation    Pointer to the CMAC operation structure.
+ * @param[in]     input        Pointer to the input data buffer.
+ * @param[in]     input_length Length of the input data in bytes.
+ *
+ * @return PSA_SUCCESS on success, or an appropriate error code on failure.
```

Comment:
nit: we use the wording `PSA_SUCCESS on success, a valid  PSA status code otherwise.`
for most of these return values and it might be nice to try to standardize our wording throughout the driver